tasks/43_sudo.yml

```diff
@@ -20,3 +20,29 @@
     create: 'yes'
   tags:
     - sudo
+
+# Taken from:
+# https://github.com/konstruktoid/ansible-role-hardening/blob/f21d1b8ac18be3f275d6fed831124aa66b0b9baf/tasks/sudo.yml#L88-L107
+#
+# This code gets rid of a compliance scan warning related to
```

Comment:
Does the warning actually go away?  If not, I suggest this:
```suggestion
# This code addresses a compliance scan warning related to
```

Reply:
Good point.